src/app/shared-component/wizard/wizard-navigator.ts

```diff
@@ -0,0 +1,21 @@
+import { WizardStepConfig } from './wizard-step-config';
+import { Wizard } from './wizard';
+
+// The navigator is used to inject wizard navigation into components and allow them to participate
+// in wizard navigation
+export class WizardNavigator {
```

Comment:
@joshuawilson probably a good idea to move the wizard over, agreed. Suggest we merge this PR and then do that? (file an issue for the move).